clouddriver-cloudfoundry/src/test/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/converters/DeployCloudFoundryServerGroupAtomicOperationConverterTest.java

```diff
@@ -102,131 +102,6 @@ void initializeClassUnderTest() {
     converter.setObjectMapper(new ObjectMapper());
   }
 
-  @Test
```

Comment:
Why aren't these tests useful anymore? The class and methods being tested still exist even though they have been trimmed down.

Reply:
It tests manifest parsing that has moved to Orca now.